ArduPlane/Parameters.cpp

```diff
@@ -1058,10 +1058,26 @@ const AP_Param::Info Plane::var_info[] = {
     GOBJECT(relay,                  "RELAY_", AP_Relay),
 
 #if PARACHUTE == ENABLED
-	// @Group: CHUTE_
+    // @Group: CHUTE_
     // @Path: ../libraries/AP_Parachute/AP_Parachute.cpp
     GOBJECT(parachute,		"CHUTE_", AP_Parachute),
 
+    // @Param: CHUTE_AUTO_ON
+    // @DisplayName: Parachute automatic emergency release
+    // @Description: Parachute automatic emergency release enabled or disabled.
+    // @Values: 0:Disabled,1:Enabled
+    // @User: Standard
+    GSCALAR(parachute_auto_enabled,    "CHUTE_AUTO_ON",   0),
```

Comment:
Move these to line 54 of this
https://github.com/diydrones/ardupilot/blob/master/libraries/AP_Parachute/AP_Parachute.cpp